ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml

```diff
@@ -95,7 +95,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          pilot 'TestGatewayConformance|TestCustomGateway|TestCNIRaceRepair|TestGateway/managed-owner|TestProxyHeaders|TestCustomGateway/helm|TestCustomGateway/helm-simple|TestTcpProbe/rewrite-success|TestGRPCProbe|TestTunnelingOutboundTraffic|TestCNIVersionSkew|TestGateway|TestIngress|TestLabelChanges|TestLocality|TestMirroring/mirror-percent-absent|TestMirroring/mirror-50|TestMirroring/mirror-10|TestMirroringExternalService'"
+          pilot 'TestGatewayConformance|TestCNIRaceRepair|TestCNIVersionSkew'"
```

Comment:
The [PR which addresses](https://github.com/istio/istio/pull/52974) `TestCustomGateway` is under review on `master` branch. So, lets skip them until its merged.

i.e., 
```
TestCustomGateway
TestCustomGateway/helm
TestCustomGateway/helm-simple
```

In my testing, I found the following tests as flaky. Would be better to skip them for now.

```
TestMirroring/mirror-50
TestMirroring/mirror-10
```

`TestMirroring/mirror-percent-absent` and `TestMirroringExternalService` was passing in my tests.